### PR TITLE
Make `--output-directory` default to pwd

### LINKED ISSUE #230: `sssom split`: Handling cases without `--output-directory`
## Description
During my meeting w/ Nico today, I suggested that the `--output-directory` flag should be optional.

## Current behavior
Right now, it is currently erroring out if this flag is not present:

```
sssom split omim.sssom.tsv

WARNING:root:No License provided, using http://w3id.org/sssom/license/unspecified
WARNING:root:No prefix map provided (not recommended), trying to use defaults..

Traceback (most recent call last):
File "/Users/joeflack4/virtualenvs/omim/bin/sssom", line 8, in <module>
sys.exit(main())
File "/Users/joeflack4/virtualenvs/omim/lib/python3.9/site-packages/click/[core.py](http://core.py/)", line 1130, in __call__
return self.main(*args, **kwargs)
File "/Users/joeflack4/virtualenvs/omim/lib/python3.9/site-packages/click/[core.py](http://core.py/)", line 1055, in main
rv = self.invoke(ctx)
File "/Users/joeflack4/virtualenvs/omim/lib/python3.9/site-packages/click/[core.py](http://core.py/)", line 1657, in invoke
return _process_result(sub_ctx.command.invoke(sub_ctx))
File "/Users/joeflack4/virtualenvs/omim/lib/python3.9/site-packages/click/[core.py](http://core.py/)", line 1404, in invoke
return ctx.invoke(self.callback, **ctx.params)
File "/Users/joeflack4/virtualenvs/omim/lib/python3.9/site-packages/click/[core.py](http://core.py/)", line 760, in invoke
return __callback(*args, **kwargs)
File "/Users/joeflack4/virtualenvs/omim/lib/python3.9/site-packages/sssom/[cli.py](http://cli.py/)", line 177, in split
split_file(input_path=input, output_directory=output_directory)
File "/Users/joeflack4/virtualenvs/omim/lib/python3.9/site-packages/sssom/[io.py](http://io.py/)", line 94, in split_file
write_tables(splitted, output_directory)
File "/Users/joeflack4/virtualenvs/omim/lib/python3.9/site-packages/sssom/[writers.py](http://writers.py/)", line 314, in write_tables
output_dir = Path(output_dir).resolve()
File "/usr/local/Cellar/python@3.9/3.9.9/Frameworks/Python.framework/Versions/3.9/lib/python3.9/[pathlib.py](http://pathlib.py/)", line 1082, in __new__
self = cls._from_parts(args, init=False)
File "/usr/local/Cellar/python@3.9/3.9.9/Frameworks/Python.framework/Versions/3.9/lib/python3.9/[pathlib.py](http://pathlib.py/)", line 707, in _from_parts
drv, root, parts = self._parse_args(args)
File "/usr/local/Cellar/python@3.9/3.9.9/Frameworks/Python.framework/Versions/3.9/lib/python3.9/[pathlib.py](http://pathlib.py/)", line 691, in _parse_args
a = os.fspath(a)

TypeError: expected str, bytes or os.PathLike object, not NoneType
```

## Decided behavior
After brief discussion, we agreed on this, and decided that the default behavior in this case would be that the output directory should be the present working directory (`pwd`) that `sssom` is being called from.

I think that if you `import os` and use `os.getcwd()`, this should return the path of the user's `pwd`.

### PR DESCRIPTION
**Pull request by hrshdhgd**
Fixes #230 

For now only implemented in `sssom split abc.sssom.tsv`

### CODE REVIEW

File: sssom/cli.py [hunk 1]
```diff
@@ -172,7 +173,13 @@ def validate(input: str):
 
 @main.command()
 @input_argument
-@output_directory_option
+@click.option(
```
Comment (matentzn) [inline sssom/cli.py:176]: why not instead at the `default=os.getcwd()` to the `@output_directory_option` above? should be useful for partition as well!

Comment (hrshdhgd) [reply sssom/cli.py:176]: Hence [the question in the ticket](https://github.com/mapping-commons/sssom-py/issues/230#issuecomment-1101780757).

Comment (hrshdhgd) [reply sssom/cli.py]: Done.

**Review by matentzn — APPROVED** [review]

File: sssom/cli.py (hunk 1 above)
Comment (matentzn) [reply sssom/cli.py]: lovely

**Review by joeflack4 — COMMENTED** [review]
Pretty succinct change! Thanks.

**Pull request merged**